hudi-utilities/src/main/java/org/apache/hudi/utilities/multitable/MultiTableServiceUtils.java

```diff
@@ -166,6 +168,9 @@ public static TableServicePipeline buildTableServicePipeline(JavaSparkContext js
                                                                HoodieMultiTableServicesMain.Config cfg,
                                                                TypedProperties props) {
     TableServicePipeline pipeline = new TableServicePipeline();
+    HoodieTableMetaClient metaClient = UtilHelpers.createMetaClient(jsc, basePath, true);
+    // Add the table config to the write config.
+    props.putAll(metaClient.getTableConfig().getProps());
```

Comment:
Should we use table config to overwrite those configs already set in write configs by user?  Not sure which one have a higher priority here. @danny0405 What do you think?

Reply:
This can indeed lead to a priority issue here. A simple solution is to use `addNecessaryTableConfigToWriteConfig` to add the necessary parameters.

Reply:
Are you refering to the per-table options? We should consider write config options with higher priority always.